mesheryctl/internal/cli/root/root_test.go

```diff
@@ -0,0 +1,86 @@
+package root
+
+import (
+	"bytes"
+	"fmt"
+	"path/filepath"
+	"runtime"
+	"testing"
+
+	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
+	"github.com/sirupsen/logrus"
+)
+
+type RootCmdTestInput struct {
+	Name             string
+	Args             []string
+	ExpectedResponse string
+}
+
+func TestRootCmdIntegration(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping integration test")
+	}
+
+	// get current directory
+	_, filename, _, ok := runtime.Caller(0)
+	if !ok {
+		t.Fatal("Not able to get current working directory")
+	}
+	currDir := filepath.Dir(filename)
+
+	// update all locations
+	fixturesDirPath := filepath.Join(currDir, "fixtures")
+	utils.MesheryFolder = filepath.Join(fixturesDirPath, ".meshery")
+	utils.DefaultConfigPath = filepath.Join(utils.MesheryFolder, "config.yaml")
+	testConfigPath := filepath.Join(fixturesDirPath, "testConfig.yaml")
+
+	tests := []RootCmdTestInput{
+		{
+			Name: "Gets version",
+			Args: []string{"version"},
+		},
+		{
+			Name:             "view the channel to test if meshconfig is correctly created & loaded",
+			Args:             []string{"system", "channel", "view"},
```

Comment:
Used this command i.e `mesheryctl system channel view` to test configs as it's output is simple and short. Any other better alternatives?

Reply:
Good one. This is as short and sweet as it gets, I believe.